string channel_name                         # User defined name
mechanism_msgs/JointStates joint_states     # The settled pixel location
